src/components/container/list/list.module.css

```diff
@@ -0,0 +1,45 @@
+.list{
+    margin: auto;
+    margin-top: 30px;
+    margin-bottom: 20px;
+    min-height: 600px;
+    width: 400px;
+    background-color: lavender;
+    border-radius: 20px;
+    text-align: left;
+    padding-left: 20px;
+    padding-top: 20px;
+    font-family: 'Kalam', cursive;
+    color: darkslategray;
+    box-shadow: 5px 5px 10px darkslategray;
+    list-style: none;
+
+}
+.complete{
+    border: 2px solid mediumseagreen;
+    background-color: white;
+    height: 35px;
+    width: 35px;
+    margin-bottom: 10px;
+    color:mediumseagreen;
+    margin-right: 5px;
+}
+.complete:hover{
+    background-color:mediumseagreen;
+    color: white;
+}
+.delete{
+    border: 2px solid coral;
+    background-color: white;
```

Comment:
style powtarzające się , tak jak tutaj na przykład masz białe tło dla przycisku delete i complete możesz dać do głównego folderu SRC do pliku ze zmiennymi na przykład tworzysz folder Styles -> plik : variable.js i tam :
export const colors = {
buttonBackground:"#FFFFFF"
}
to importujesz do list.module.css i do klasy complete w background-color podstawiasz colors.buttonBackground